fortex/health/processors/xray_processor.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2022 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+XRAY_Processor Processor
+"""
+from typing import Dict, Set
+import PIL
+from transformers import pipeline
+from forte.data.data_pack import DataPack
+from forte.common.resources import Resources
+from forte.common.configuration import Config
+from forte.processors.base import PackProcessor
+from ft.onto.base_ontology import Classification
+
+
+__all__ = [
+    "XRAY_Processor",
+]
+
+
+class XRAY_Processor(PackProcessor):
```

Comment:
As you might have observed from our existing code base, we name the classes in Pascal case. Lets remove the snake case, and also make sure its consistent with the `XrayImageReader`.